cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/Database200MigrationTest.java

```diff
@@ -100,7 +102,7 @@ public Void call(SQLDatabase db) throws Exception {
 
 
         int count = revisionCount();
-        Assert.assertThat(count, is(2));
+        Assert.assertThat("That a duplicate root has been removed.", count, is(2));
```

Comment:
If anything the failure message in the report is more important as it is, more often than not, read in isolation. On the other hand whilst reading code the surrounding code and nearby javadoc etc provides some additional context that is not available in the report.
You don't have to follow my advice about using "should", but I would like the messages to at least make sense in a failure case.